pandas/tests/test_resample.py

```diff
@@ -3416,3 +3416,10 @@ def test_aggregate_with_nat(self):
 
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
+
+    def test_repr(self):
+        result = repr(TimeGrouper(key='A', freq='H'))
+        expected = ("TimeGrouper(key='A', freq=<Hour>, axis=0, sort=True, "
+                    "closed='left', label='left', how='mean', "
```

Comment:
can you add a similar test for Grouper (in test_groupby)

Reply:
added